systers_portal/templates/meetup/members.html

```diff
@@ -0,0 +1,22 @@
+{% extends "meetup/base.html" %}
+
+{% block meetup_location_page_content %}
+<div class="box-container">
+  <h3>Organizers</h3>
+  <ul>
+   {% for organizer in object.organizers.all %}       
+      <li>
+        <a href="{{ organizer.get_absolute_url }}">{{ organizer }}</a> 
```

Comment:
It would be nice to have the user profile pics as well, arranged maybe in a grid. Could you mockup this?